### DISCUSSION TITLE
Devices compatibility

### DISCUSSION THREAD

**Discussion by mikkk3l** [gmallios/SoundcoreManager#142] [General]
Hi, I would appreciate you if you could make it compatible wih Soundcore Life P3. Would it be possible? Thank you very much in advance for your fantastic work!

**Comment by gmallios** [comment]
> Hi, I would appreciate you if you could make it compatible wih Soundcore Life P3. Would it be possible? Thank you very much in advance for your fantastic work!

Yes, it's possible. I had a quick look and it seems like it won't take long but I don't have the device on-hand and I will need someone to test (possibly multiple versions). I will tag you in an issue when the time comes!

**Reply by JohnRigoni** [reply]
Hello, I have the P3 and would be happy to test. Would also be happy to do the work myself, is there any documentation on how to reverse engineer the device/contribute? Thanks.
They just added unskippable ads to the mobile app so I am motivated.

**Comment by artemererer** [comment]
I'm looking for the VR P10 compatibility 'cause it's made specially for gaming and it would be very nice to have access to options on pc, either via dongle or bluetooth.

**Comment by dalivaktuk** [comment]
I would like to join in testing for the A3033 Life 2 Neo headsets as I have them and would you also be able to add the Life Note 3i?

**Comment by Panabou** [comment]
Hello, this is a great project! If there is a chance to also support the Soundcore Liberty 4 headphones, I would like to help with the testing.
